TheraLang.Web/ClientApp/src/app/modules/paginationg/pagination.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges} from '@angular/core';
+import { TranslateService } from "@ngx-translate/core";
+import { Router, ActivatedRoute } from "@angular/router";
+import {MatDialog} from "@angular/material";
+
+@Component({
+  selector: "app-pagination",
+  templateUrl: "./pagination.component.html",
+  styleUrls: ["./pagination.component.less"]
+})
+
+export class PaginationComponent implements OnInit, OnChanges {
+  @Input() collection: any[];
```

Comment:
Why not send only collection size instead of all collection?

Reply:
collection size its total number of pages on backend to know total number of page, but collection [] its how elements in one page